metadata-ingestion/tests/unit/api/test_circuit_breaker.py

```diff
@@ -0,0 +1,66 @@
+import json
+from unittest.mock import patch
+
+from datahub.api.circuit_breaker import (
```

Comment:
I noticed we are not testing Operation circuit breaker? 

Let's try to achieve testing parity between the two.